charts/prometheus-blackbox-exporter/templates/deployment.yaml

```diff
@@ -21,6 +21,9 @@ spec:
     metadata:
       labels:
         {{- include "prometheus-blackbox-exporter.selectorLabels" . | nindent 8 }}
+        {{- if .Values.commonLabels }}
+{{ toYaml .Values.commonLabels | indent 8 }}
+        {{- end }}
```

Comment:
this should not be necessary: https://github.com/es1o/helm-charts/blob/common-labels/charts/prometheus-blackbox-exporter/templates/_helpers.tpl#L46-L48

Reply:
@monotek It is necessary because `prometheus-blackbox-exporter.labels` are not used here, only `prometheus-blackbox-exporter.selectorLabels` that doesn't have `.Values.commonLabels`.